modules/dynamic-plugins/proc-install-plugins-using-custom-npm-registry.adoc

```diff
@@ -5,16 +5,16 @@
 
 You can install external plugins in an air-gapped environment by setting up a custom NPM registry.
 
-You can configure the NPM registry URL and authentication information for dynamic plugin packages using a Helm chart. For dynamic plugin packages obtained through `npm pack`, you can use a `.npmrc` file. 
+You can configure the NPM registry URL and authentication information for dynamic plugin packages using a Helm chart. For dynamic plugin packages obtained through `npm pack`, you can use a `.npmrc` file.
 
-Using the Helm chart, add the `.npmrc` file to the NPM registry by creating a secret named `dynamic-plugins-npmrc` with the following content:
+Using the Helm chart, add the `.npmrc` file to the NPM registry by creating a secret. For example:
 
-[source,yaml]
+[source,yaml,subs="+quotes,+attributes"]
 ----
 apiVersion: v1
 kind: Secret
 metadata:
-  name: dynamic-plugins-npmrc
+  name: `_<release_name>_-dynamic-plugins-npmrc`
```

Comment:
Do you want to also add a note explaining that `<release_name>` should be replaced? Just like what you wrote in https://github.com/redhat-developer/red-hat-developers-documentation-rhdh/pull/962/files#diff-20d4572b8b8466c90fde790084a52a4c3ca7ea49241bf26c509c224fa29c1a6fR61?